_blog.yml

```
@@ -4782,3 +4782,13 @@
     - gradio
     - spaces
     - open-source
+
+- local: keras_llama_32
```

```suggestion
- local: keras-llama-32
```
we tend to use dashes instead of underscores in URLs

---

done